content/docs/rfcs/57/README.md

```diff
@@ -0,0 +1,338 @@
+  // A mapping of chats to their details
+  map<string,CommunityChat> chats = 6;
+  // A list of banned members
+  repeated string ban_list = 7;
+  // A mapping of categories to their details
+  map<string,CommunityCategory> categories = 8;
+  // The admin settings of the Community
+  CommunityAdminSettings admin_settings = 10;
+  // If the community is encrypted
+  bool encrypted = 13;
+  // The list of tags
+  repeated string tags = 14;
+  // index of the community's shard within the Status shard cluster
+  uint16 relay_shard_index = 15
+}
+```
+
+> *Note*: Currently, Status app has allocated shared cluster `16` in [52/WAKU2-RELAY-STATIC-SHARD-ALLOC](/spec/52/).
+Status app could allocate more shard clusters, for instance to establish a test net.
+We could add the shard cluster index to the community description as well.
+The recommendation for now is to keep it as a configuration option of the Status app.
+
+> *Note*: Once this RFC moves forward, the new community description protobuf fields should be mentioned in [56/STATUS-COMMUNITIES](https://rfc.vac.dev/spec/56/).
+
+Status communities can be mapped to shards in two ways: static, and owner-based.
+
+### Static Mapping
+
+With static mapping, communities are assigned a specific shard index within the Status shard cluster.
+This mapping is similar in nature to the shard cluster allocation in [52/WAKU2-RELAY-STATIC-SHARD-ALLOC](/spec/52/).
+Shard indices allocated in that way are in the range `16 - 127`.
+The Status CC community uses index `16` (not to confuse with shard cluster index `16`, which is the Status shard cluster).
+
+### Owner Mapping
+
+> *Note*: This way of mapping will be specified post-MVP.
+
+Community owners can choose to map their communities to any shard within the index range `128 - 767`.
+
+## 1:1 Chat
+
+[55/STATUS-1TO1-CHAT](/spec/55) uses partitioned topics to map 1:1 chats to a set of 5000 content topics.
+This document extends this mapping to 8192 content topics that are, in turn, mapped to 128 shards in the index range of `768 - 895`.
+
+```
+contentPartitionsNum = 8192
+contentPartition = mod(publicKey, contentPartitionsNum)
+partitionContentTopic = "contact-discovery-" + contentPartition
+
+partitionContentTopic = keccak256(partitionContentTopic)
+
+shardNum = 128
+shardIndex = 768 + mod(publicKey, shardNum)
+
+```
+
+# Infrastructure Nodes
+
+As described in [30/ADAPTIVE-NODES](/spec/30/),
+Waku supports a continuum of node types with respect to available resources.
+Infrastructure nodes are powerful nodes that have a high bandwidth connection and a high up-time.
+
+This document, which informs about simple ways of scaling Status over Waku,
+assumes the presence of a set of such infrastructure nodes in each shard.
+Infrastructure nodes are especially important for providing connectivity in the roll-out phase.
+
+Infrastructure nodes are not limited to Status fleets, or nodes run by community owners.
+Anybody can run infrastructure nodes.
```

Comment:
In the context of Status MVP, to ensure good UX, one need will need to run a node with postgres and some minimal requirements. Should we mention this in the RFC?